wandb/plots/utils.py

```diff
@@ -0,0 +1,120 @@
+import wandb
+from wandb import util
+
+# FIXME: look at wandb.util.is_numpy_array() wandb.util.is_pandas* for examples
+np = util.get_module("numpy", required="Logging plots requires numpy")
+pd = util.get_module("pandas", required="Logging dataframes requires pandas")
+scipy = util.get_module("scipy", required="Logging scipy matrices requires scipy")
+spacy = util.get_module("spacy", required="Logging NER and POD requires spacy")
```

Comment:
Again, this will make it so that anyone that imports wandb has these libaries installed regardless of if they use plots.